Description: Run notify-send in background, to not delay daemon startup
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Bug-Debian: https://bugs.debian.org/880594
Bug-Ubuntu: https://launchpad.net/bugs/1722745
Forwarded: yes (irc)
Last-Update: 2018-03-16

--- virtualbox-5.2.8-dfsg.orig/src/VBox/Additions/x11/Installer/98vboxadd-xclient
+++ virtualbox-5.2.8-dfsg/src/VBox/Additions/x11/Installer/98vboxadd-xclient
@@ -23,7 +23,7 @@ done
 
 if ! test -c /dev/vboxguest 2>/dev/null; then
    # Do not start if the kernel module is not present.
-   notify-send "VBoxClient: the VirtualBox kernel service is not running.  Exiting."
+   notify-send "VBoxClient: the VirtualBox kernel service is not running.  Exiting." &
 elif test -z "${SSH_CONNECTION}"; then
    # This script can also be triggered by a connection over SSH, which is not
    # what we had in mind, so we do not start VBoxClient in that case.  We do
